demos/howtos/posterior.py

```diff
@@ -0,0 +1,178 @@
+"""
+How to define a Posterior distribution
```

Comment:
Great howto with gradually increasing complexity.  For naming of the file, "posterior.py" could suggest it being a posterior class or module, and does not describe what the howto does with a posterior. I suggest (in general I suppose)  including a verb in the filename, such as "defining_posteriors", "setting_up_a_posterior", or similar.